python/grass/jupyter/setup.funcs.py

```diff
@@ -0,0 +1,61 @@
+import os
```

Comment:
Run Pylint, so it tells you you should put a docstring here describing what is in this file (one sentence, it may appear in online documentation, aimed at users of the functions in the file).

Also at the very begging, please add file header with license:

https://trac.osgeo.org/grass/wiki/Submitting/Python
https://trac.osgeo.org/grass/wiki/Submitting/C

I'm little torn between using `#` comments and a docsting for that statement, but given that the docsting is meant for user documentation, `#` comments seem more appropriate for the header information which is aiming at maintainers/authors.